src/test/fuzz/transaction.cpp

```diff
@@ -103,6 +103,6 @@ FUZZ_TARGET_INIT(transaction, initialize_transaction)
     (void)IsWitnessStandard(tx, coins_view_cache);
 
     UniValue u(UniValue::VOBJ);
-    TxToUniv(tx, /* hashBlock */ uint256::ZERO, u);
-    TxToUniv(tx, /* hashBlock */ uint256::ONE, u);
+    TxToUniv(tx, /* hashBlock */ uint256::ZERO, /* entry */ u);
```

Comment:
I think you can drop one of these statements to make the fuzz test run ~twice as fast.

(In a separate commit, obviously)

Reply:
Good point. I don't see any need to exercise this code path from the fuzz test, which seems to be the only point of invoking `TxToUniv` again w/ `/* hashBlock */ uint256::ONE` (especially if it will run twice as fast)

```c++
if (!hashBlock.IsNull())
  entry.pushKV("blockhash", hashBlock.GetHex());
```

So I'll add a commit removing `TxToUniv(tx, /* hashBlock */ uint256::ONE, u);` as suggested.

Reply:
Done in this commit c45b13389a3540a07c0f64f63cb36f538f8e2f1c

Reply:
Can also be done in a follow-up